src/inlineSuggestions/snippets/snippetDecoration.ts

```diff
@@ -84,9 +84,28 @@ async function insertBlankSnippet(
 
 export function handleClearSnippetDecoration(): void {
   if (snippetBlankRange) {
+    const fixedRange = calculateStartAfterUserInput(snippetBlankRange);
+
+    if (fixedRange) snippetBlankRange = fixedRange;
```

Comment:
no way, how  did the linter allowed you to write this bizarre line ? 🤣

Reply:
Will change